sim/warrior/arms/talents.go

```diff
@@ -59,6 +60,7 @@ func (war *ArmsWarrior) applyTasteForBlood() {
 		ClassSpellMask: warrior.SpellMaskRend,
 		ICD:            5 * time.Second,
```

Comment:
This ICD is 5s for the proc but the application aura is 6s. This looks weird to me